Source/BSN.IpTables.Cli/custom/Module.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace BSN.IpTables.V1
+{
+    public partial class Module
```

Comment:
I suggest add some technical comments (implementation, not API comment), to describe what is Module? and how to work with that